app/views/components/gds/gdsMainTemplate.scala.html

```diff
@@ -44,7 +44,8 @@
 @(
   title: Title,
   backButton: Option[BackButton] = None,
-  useCustomContentWidth: Boolean = false
+  useCustomContentWidth: Boolean = false,
+  navigationBanner: Html = HtmlFormat.empty
```

Comment:
Nit: It's not that important, but you can put not only navigation banner here. You can put anything that needs to go before the page's main content.